packages/ketcher-core/src/application/render/renderers/PolymerBondRenderer/SnakeModePolymerBondRenderer.ts

```diff
@@ -130,7 +130,7 @@ export class SnakeModePolymerBondRenderer extends BaseRenderer {
       .append('path')
       .attr('stroke', this.polymerBond.finished ? '#333333' : '#0097A8')
       .attr('stroke-width', 1)
-      .attr('class', 'selection-area')
+      .attr('class', 'selection-area snake-bond')
```

Comment:
Perhaps this change is not needed then too?

Reply:
Fix